src/fromager/packagesettings.py

```diff
@@ -371,6 +376,12 @@ class PackageSettings(pydantic.BaseModel):
     project_override: ProjectOverride = Field(default_factory=ProjectOverride)
     """Patch project settings"""
 
+    exclusive_build: bool = Field(
+        default=False,
+        description="If true, this package must be built on its own (exclusive build). Default: False.",
+    )
+    """If true, this package must be built on its own (exclusive build). Default: False."""
```

Comment:
The explanation is a bit cryptic. Could you please mention that this setting influences parallel builds?

Reply:
I tried adding a bit of text, let me know if that's clearer?